src/dapps/eth-blocks/TheEthBlocksLayout.vue

```diff
@@ -70,6 +74,13 @@ export default {
       if (newVal) {
         this.setCheckPendingInterval();
       }
+    },
+    $route(to) {
+      if (to.name === ETH_BLOCKS_ROUTE.MY_BLOCKS.NAME) {
+        this.activeTab = this.tabs[1].id;
+      } else {
+        this.activeTab = this.tabs[1].id;
```

Comment:
NOTE: This should be: 
`this.activeTab = this.tabs[0].id;`
FIX is in the next commit